sample/src/main/java/com/ramcosta/destinations/sample/TempUtils.kt

```diff
@@ -0,0 +1,18 @@
+package com.ramcosta.destinations.sample
+
+import com.ramcosta.composedestinations.spec.DestinationSpec
+import com.ramcosta.composedestinations.spec.NavGraphSpec
+import com.ramcosta.composedestinations.spec.Route
+import com.ramcosta.destinations.sample.destinations.DirectionDestination
+import com.ramcosta.destinations.sample.destinations.TypedDestination
+import java.io.InvalidObjectException
+
+inline fun <reified T> Any.asRoute(): Route? {
```

Comment:
Seems to me like an unnecessary pollution of the Any class. Could we make the type more specific or move the function to some other place instead of extending Any class?

Reply:
I thought I deleted this file actually 🤔 guess I forgot it when I pushed. I'll correct this today! Thanks!

Reply:
@Nek-12 I can confirm this file was deleted in the final commit. The initial idea with this function was to convert a NavGraphSpec, DestinationSpec or String? into a Route? which as you can see by the types, have very little in common 😉 But nonetheless, thank you for pointing it out! I'll make sure to be more cautious in the future!